samples/demo/README.md

```diff
@@ -0,0 +1,15 @@
+# Hive JS SDK Demo
+
```

Comment:
Shouldn't we have all this in a folder named `ionic` to be able to add other implementation examples? We may rename `demo` to `ionic` since the root folder `samples` is already an implementation sandbox.

Reply:
ok